babel/.babelrc.json

```diff
@@ -0,0 +1,7 @@
+{
+	"presets": [
+    [
+      "@babel/preset-env"
+    ]
+  ]
+}
```

Comment:
Tabs/spaces mixed up, I guess we don't use prettier correct.

Update `format` command in `package.json` to include `.json` or better, update it to how its used in smlxl monorepo.

Reply:
Updated package.json and ran `yarn format` to take care of whatever it could.